stanford/roles/cluster/defaults/main.yml

```diff
@@ -131,61 +131,61 @@ CLUSTER_SECURITY_GROUP_RULES:
     #   from_port: 11371
```

Comment:
Can the commented block here be removed?

Reply:
Done.